ds-prep-python-loops/main.py

```diff
@@ -0,0 +1,33 @@
+record = (1, "Grimdiana", "Bones", "boulders")
+row = ""
+for x in record:
+  print(x)
+for x in record:
+  for y in row:
+    print(x , "," , y)
+values_list = [1, 1, 2, 3, 5, 8, 13, 21, 34]
+for x in range(len(values_list)):
+  print(values_list[x])
+index_list = []
+for x in values_list:
+  for y in index_list:
+    print(x , y)
+for x in index_list:
+  print(x)
+vowels = {"A", "E", "I", "O", "U"}
+parts_of_the_big_letter = {"L", "M", "N", "O", "P"}
+for x in vowels:
+  print(x)
+print(parts_of_the_big_letter.difference(vowels))
```

Comment:
Instead of using the `difference` method, try doing this manually with the `for` loop:

---

![image](https://user-images.githubusercontent.com/16143771/149633513-6a9abf46-da01-4b82-a0d6-a3ce3d69024c.png)